src/openzaak/components/documenten/api/serializers.py

```diff
@@ -210,6 +196,9 @@ def get_object(self, view_name, view_args, view_kwargs):
         except (TypeError, AttributeError):
             self.fail("does_not_exist")
 
+    def get_attribute(self, instance):
+        return instance.get_informatieobject()
+
```

Comment:
Can you explain why you did this?

Reply:
`EnkelvoudigInformatieObjectHyperlinkedRelatedField` is used for all documenten subresources like `Gebruiksrechten` for `informatieobject` field.
The custom field was needed because:
* in case CMIS is enabled CMIS is requested
* in case CMIS is disabled the subresource has FK to `EnkelvoudigInformatieObjectCanonical`, which doesn't have `uuid`, so `canonical.latest_version` property should be used. 

In the old implementation `get_url` method  was customized to solve this issue.  `get_url` is used inside `to_representation` method, which is used on `field.get_attribute` value
So how it was before:
* `Gebruiksrechten` serializer calls `field.get_attribute` and receives the fk object which is `EnkelvoudigInformatieObjectCanonical`
* `field.to_representation` is called with the value from previous step as parameter
* `field.get_url` is called inside `field.to_representation` to convert model objects to the url.

I thought that `get_url` was not the best place for this logic, because we need to change nt the url  implementation but retrieving model object. Moreover all subresources models have `get_informatieobject` which is used for permissions and already has the implementation to get the correct `informatieobject` , so I moved the logic from `get_url` to `get_attribute` method